src/module.ts

```diff
@@ -646,6 +630,17 @@ export class Module {
     }
   }
 
+  removeGlobal(
+    name: string
+  ): void {
+    var cStr = allocString(name);
+    try {
+      _BinaryenRemoveGlobal(this.ref, cStr);
```

Comment:
`try/finally` is not working for catching the `not defined` error here, you can try it in console of browser or node by yourself

```js
try {
  _BinaryenRemoveGlobal
} finally {}
```

Reply:
If you get a "not defined" kind of error there, your Binaryen version is too old. Make sure to use the one in package.json. The try-finally block is there to clean up memory in case of errors only.

Reply:
OK, but it is so weird I've got an error from `cli/asc.js` in https://travis-ci.org/JounQin/wasm-study/builds/441947604#L548, and I could reproduce it on my own machine for a long time today. However, I tried to upgrade all dependencies again just now, it worked... Maybe an issue of `npm/yarn`, still confusing.